gulp-tasks/scripts.js

```diff
@@ -0,0 +1,93 @@
+/*
+  Copyright 2016 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+/* eslint-env node */
+
+'use strict';
+
+const gulp = require('gulp');
+const uglify = require('gulp-uglify');
+const sourcemaps = require('gulp-sourcemaps');
+const browserify = require('browserify');
+const buffer = require('vinyl-buffer');
+const source = require('vinyl-source-stream');
+const path = require('path');
+const glob = require('glob');
+const eslint = require('gulp-eslint');
+
+const bundleJS = function(fullFilePath) {
```

Comment:
Quick check: this Rollup → Babel → Browserify pipeline isn't actually needed by any of the existing code, right? Are you just putting this in place with the idea that you'll rewrite the existing code to use ES2015 modules?